src/Deviantart/Provider.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace SocialiteProviders\deviantART;
+namespace SocialiteProviders\Deviantart;
```

Comment:
Should the DeviantArt classes be under a `DeviantArt` directory and namespace to match the site naming?
```suggestion
namespace SocialiteProviders\DeviantArt;
```

Reply:
I'm matching the namespace as defined in DeviantartExtendSocialite.php
Changing the namespace of that file, would probably break a couple of people who already use it: https://packagist.org/packages/socialiteproviders/deviantart

Reply:
Ah, ok. 👍

Reply:
Thanks. Let me check if there's any other major version bumps required.

Reply:
If you're doing a major version for DeviantArt, is it worth making it consistent with the actual site naming then? 🤨

Reply:
Yep, feel free :)